### PR TITLE
Adds an attribute called IgnoreProperty that tells the generator to skip the property

### PR DESCRIPTION
**Pull request by emorell96**
I added an attribute that allows the user to set some properties as ignored for more control when generating a builder.

It allows me to bypass #43 by ignoring the property that breaks the code.

I tested the attribute by adding it to the SimpleClass and the test `FluentBuilderGeneratorTests.FluentBuilderSourceGeneratorTests.GenerateFiles_ForSimpleClass_Should_GenerateCorrectFiles` should fail if the property is not skipped since the generated code wouldn't match the expected code.

The test passes, as do all the others:
image

### CODE REVIEW

**Comment by emorell96** [comment]
Ok this PR is ready. I have moved the attribute to the Attribute Generator. And it works as expected.

File: src/FluentBuilderGenerator/FileGenerators/AutoGenerateBuilderAttributeGenerator.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public FileData GenerateFile()
             Name,
             $@"//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by https://github.com/StefH/FluentBuilder.
+//     This code was generated by https://github.com/StefH/FluentBuilder version {System.Reflection.Assembly.GetExecutingAssembly().GetName().Version}
```
Comment (StefH) [inline src/FluentBuilderGenerator/FileGenerators/AutoGenerateBuilderAttributeGenerator.cs:29]: Nice addition.

Can you add this to all generated classes?

File: src/FluentBuilderGenerator/FileGenerators/FluentBuilderClassesGenerator.cs [hunk 2]
```diff
@@ -223,12 +223,13 @@ private static StringBuilder GenerateWithPropertyActionMethod(ClassSymbol classS
             .AppendLine($"            return {cast}builder.Build(useObjectInitializer);")
             .AppendLine("        });");
     }
-
+    private const string IgnoreAttributeClassName = "FluentBuilder.IgnorePropertyAttribute";
```
Comment (StefH) [inline src/FluentBuilderGenerator/FileGenerators/FluentBuilderClassesGenerator.cs:226]: move const to top of the file

**Review by StefH — CHANGES_REQUESTED** [review]

**Comment by StefH** [comment]
@emorell96 
thank you.

please see my code review

**Comment by emorell96** [comment]
I'll get to them during the weekend :) I think they are a good idea.

**Comment by StefH** [comment]
@emorell96 Did you have time yet to fix the comments?

**Comment by StefH** [comment]
@emorell96 If you don't have the time, that's also fine. I can take over your branch/PR and update it.

**Pull request merged**